Product/Production/Common/Properties/src/main/resources/gateway.properties

```diff
@@ -110,8 +109,9 @@ CoreX12GenericBatchTimeStampTimeToLive=300
 # This is only valid for  Core X12 Batch request and response services.
 CoreX12GenericBatchFutureTimeToLive=60
 
-#           Direct Message Monitoring properties
-#   ****These properties should be moved to DIRECT Config Service setting****
+!           Direct Message Monitoring properties
+!   ****These properties should be moved to DIRECT Config Service setting****
```

Comment:
For display purposes.  I filter out "!" from displayed comments for each property value.  The admin gui uses the "#" values for the description of the properties.  Most of the "!" values are either commented out properties or group descriptions.